src/event/alias.js

```diff
@@ -19,24 +19,6 @@ jQuery.each( ( "blur focus focusin focusout resize scroll click dblclick " +
 jQuery.fn.extend( {
 	hover: function( fnOver, fnOut ) {
```

Comment:
In case you haven't heard, [I'm not a fan of this method](http://blog.methvin.com/2011/10/this-hover-cruft-is-full-of-ills.html).